CMakeLists.txt

```diff
@@ -311,6 +310,11 @@ set (INCLUDE_DIRECTORIES ${INCLUDE_DIRECTORIES} ${VJSON_INCLUDE_DIRS})
 find_package (LibArchive REQUIRED)
 set (INCLUDE_DIRECTORIES ${INCLUDE_DIRECTORIES} ${LibArchive_INCLUDE_DIRS})
 
+if (MACOSX)
```

Comment:
Is this necessary?

Reply:
I don't recognize these lines. Does it compile without?

Reply:
The mac compiles happily without!